R/utilities-configuration-plan.R

```diff
@@ -117,19 +119,58 @@ getCurvePropertiesForTimeProfiles <- function(configurationPlanCurve,
 
   # Observed Data
   if (pathArray[2] %in% "ObservedData") {
-    observedData <- getObservedDataFromConfigurationPlan(pathArray[1], configurationPlan, logFolder)
+    observedResults <- getObservedDataFromConfigurationPlan(pathArray[1], configurationPlan, logFolder)
+
+    time <- ospsuite::toUnit(
+      quantityOrDimension = "Time",
+      values = as.numeric(observedResults$data[, 1]),
+      targetUnit = axesProperties$x$unit,
+      sourceUnit = observedResults$metaData$time$unit
+    )
+    outputValues <- ospsuite::toUnit(
+      quantityOrDimension = ospsuite::getDimensionForUnit(observedResults$metaData$output$unit),
+      values = observedResults$data[, 2],
+      targetUnit = axesProperties$y$unit,
+      sourceUnit = observedResults$metaData$output$unit,
+      molWeight = molWeight
+    )
+
+    outputUncertainty <- NULL

```

Comment:
here and everywhere: let's use the same terminology as in PK-Sim/MoBi/etc. and call it "Error" instead of "Uncertainty"